src/e2e/src/tests/trade.spec.ts

```diff
@@ -35,7 +44,8 @@ describe('UI Tests for Reactive Trader Cloud Web Application', async () => {
 
   it('Verify page title', async () => {
     const title = await browser.getTitle()
-    expect(title).toBe('Reactive Trader Cloud')
+    const isValid = envTitles.includes(title)
+    expect(isValid).toBeTruthy()
```

Comment:
You typically know which environment your test is running in, so can be very specific about what to expect rather than saying "One of all of these".